inc/functions.php

```diff
@@ -46,6 +46,9 @@ function spacious_scripts_styles_method() {
    // Add Genericons, used in the main stylesheet.
    wp_enqueue_style( 'spacious-genericons', get_template_directory_uri() . '/genericons/genericons.css', array(), '3.3.1' );
 
+   // Enqueue font-awesome style.
+   wp_enqueue_style( 'spacious-font-awesome', get_template_directory_uri() . '/font-awesome/css/font-awesome.min.css', array(), '4.7.0' );
```

Comment:
You seems to have not added the font-awesome library within the theme. So, please ensure that you have also added the font-awesome library too since it is enqueued.